docs/source/flow_setup.rst

```diff
@@ -267,33 +304,29 @@ You can test your installation by running
     python examples/train.py singleagent_ring --rl_trainer Stable-Baselines
 
 
+(Optional) Install h-baselines
+------------------------------
 
-(Optional) Visualizing with Tensorboard
----------------------------------------
-
-To visualize the training progress:
+h-baselines is another variant of OpenAI's baselines that support the use of
+single-agent, multiagent, and hierarchical policies. To install h-baselines,
```

Comment:
this builds off stable baselines and not off baselines right? They should be credited then

Reply:
i'll switch to stable-baselines